src/templates/prototypes/forms.pug

```diff
@@ -21,7 +19,7 @@ block content
 
           form(action='postcode')
             .form-group
-              +input('Search by postcode, for example; SW1A 0AA', 'postcode_main', 8, '[0-9a-zA-Z ]{5,}')
+              +input('Search by postcode, for example; SW1A 0AA', 'postcode_main', 'text', 8, '[0-9a-zA-Z ]{5,}', '- Special characters (!@£$%) are not permitted')
```

Comment:
Sure, which is fine, but that message really doesn't make sense. What if I enter an asterisk (*), or a question mark(?). Listing only `(!@£$%)` wouldn't make sense. It'd make sense to change this to "Only alphanumeric characters (A-Z or 0-9) allowed" at the moment.